test/utilities/fileSystem.ts

```diff
@@ -5,22 +5,17 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-import { ChildProcess, exec } from 'child_process';
 import * as fs from 'fs-extra';
 import path from 'path';
 import { TestSetup } from '../testSetup';
 import { log } from './miscellaneous';
 
-export function createFolder(folderPath: string): ChildProcess {
-  const childProcess = exec(`mkdir "${folderPath}"`);
-
-  return childProcess;
+export function createFolder(folderPath: string): Promise<void> {
+  return fs.mkdirp(folderPath);
```

Comment:
Good find!  The APIs in the `fs` module are probably faster than `exec`.  I'm trying to recall if there was a reason why I used `exec` and `ChildProcess` but I can't think of any.  Well, if running all of the suites pass, then I say let's switch.

Reply:
This gets us closer to platform independence (thinking Windows here)